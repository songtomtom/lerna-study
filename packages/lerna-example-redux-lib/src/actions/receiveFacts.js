export const RECEIVE_FACTS = 'RECEIVE_FACTS'

export function receiveFacts(facts) {
  return {
    type: RECEIVE_FACTS,
    facts
  }
}
