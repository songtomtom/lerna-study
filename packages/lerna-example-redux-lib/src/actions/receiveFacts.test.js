import { receiveFacts } from './receiveFacts'

test('returns correct value', () => {
  expect(receiveFacts(['test one', 'test two'])).toMatchSnapshot();
})
