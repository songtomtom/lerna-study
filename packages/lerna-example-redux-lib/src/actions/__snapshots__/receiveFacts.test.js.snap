// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct value 1`] = `
Object {
  "facts": Array [
    "test one",
    "test two",
  ],
  "type": "RECEIVE_FACTS",
}
`;
